### DISCUSSION TITLE
Intentionally cause a Red Screen Error for training purposes

### DISCUSSION THREAD

**Discussion by uq-itali** [SafeExamBrowser/seb-mac#211] [Ideas]
Hello,

I'm working at an institution using SEB for exams and we're training new staff in problems encountered when using it. A relatively common problem at each exam is about 1-2% of the cohort having the Red Screen error. We want to cause a red screen error consistently so that we can train new staff in how to react and resolve this issue.

Is there any full proof ways to cause a Red Screen error consistently?

I find that once I resolve a red screen error on my own mac device for background apps being open or an external display being connected this does not work a second time to cause the same error.

Thanks

Tim

**Comment by danschlet** [comment]
Hi Tim,

There are many possible reasons such a red lock screen can be displayed. The easiest to reproduce is to change the macOS user if you have more than one user account on a machine (it's enough to go into the select user login screen by first invoking sleep mode). You can also invoke a lock screen trying to connect with screen sharing or remote management (if enabled in System Preferences / Sharing) from another machine. Regarding the lock screen for background apps, there was a bug in SEB 3.0, which is fixed in 3.1 (see [release candidate](https://github.com/SafeExamBrowser/seb-mac/releases/tag/3.1rc), the final version should be out this week). Also the lock screen for Siri or dictation might be easy to invoke if you look up the key short cuts for these features.

Daniel

**Reply by uq-itali** [reply]
Thanks Daniel. I'll give all these a try and report back
